handbook/engineering/search/index.md

```diff
@@ -21,7 +21,7 @@ Our current goals are documented in the [search team's current tracking issue](h
 
 ## On-call
 
-- [Alerts owned by this team](https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24+file:monitoring/.*+%7B:%5B_%5D%2C+Owner:+ObservableOwnerSearch%2C+:%5B_%5D%7D&patternType=structural)
+- [Alerts owned by this team](https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24+file:monitoring/.*+ObservableOwnerSearch&patternType=literal)
```

Comment:
why the literal search instead of the structural one? I figured it might be useful to see the names and other details of each alert in the result snippets as well

Reply:
both queries still work for me:

* [`repo:^github.com/sourcegraph/sourcegraph$ file:monitoring/.* {:[_], Owner: ObservableOwnerCloud, :[_]}`](https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24+file:monitoring/.*+%7B:%5B_%5D%2C+Owner:+ObservableOwnerCloud%2C+:%5B_%5D%7D&patternType=structural)
* [`repo:^github.com/sourcegraph/sourcegraph$ file:monitoring/.* {:[_], Owner: ObservableOwnerSearch, :[_]}`](https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24+file:monitoring/.*+%7B:%5B_%5D%2C+Owner:+ObservableOwnerSearch%2C+:%5B_%5D%7D&patternType=structural)

Reply:
Because the results are incomplete in that way, due to the refactoring we did to parameterise owners in reused functions that produce alerts (Observables).

Reply:
would this work? https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24+file:monitoring/.*+%7B:%5B_%5D%2C+Owner:+ObservableOwnerCloud%2C+:%5B_%5D%7D+OR+%28:%5B_%5D%2C+ObservableOwnerCloud%29&patternType=structural

Reply:
I just think it's valuable to have more of the alert show in the results

Reply:
That works, thanks!